chichiwang/exercise-js
# Accessibility compliance of links and SVGs

## Description
Linked to Issue: #155
Fix WACG AA standards compliance of the homepage link in the Nav and on heading links. External links are also changed to no longer open in a new tab, as [advised by WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/G200.html)

Additionally, the Egret SVG contained an element with an ID - given that is used multiple times throughout a given document and the site, and that the ID is not ever referenced, the ID attribute is removed.

## Changes
* Specify the aria label for the homepage link in `app/Layout/Nav.tsx`
* Update `app/components/ExternalLink/index.tsx` to no longer open in a new tab
* Specify an aria-label for the link in `app/components/Heading/index.tsx` to denote that it is a link to the current section
* Remove the `id` attribute from the path in the SVG in `app/components/Logo/Egret.tsx`

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no errors
* Ensure that no links open in a new tab by default in the browser

## After merge
* Deploy this change to production and run an [accessibility test](https://accessibilitytest.org/) on it to ensure no major issues remain
